compiler-core/src/build/package_compiler.rs

```diff
@@ -279,7 +302,7 @@ where
         Ok(())
     }
 
-    fn encode_and_write_metadata(&mut self, modules: &[Module]) -> Result<()> {
+    fn encode_and_write_metadata(&mut self, modules: &Vec<Module>) -> Result<()> {
```

Comment:
How come this has changed?

Reply:
Oh, that was a remainder of some (unpublished) attempts. Will change it back!